src/pwm_sound_device.rs

```diff
@@ -0,0 +1,494 @@
+use crate::consts::*;
```

Comment:
Can this file be split into `pwm.rs` and `dma.rs`, and implement the sound device in rCore kernel?